### DISCUSSION TITLE
ScriptReference/Application-isEditor

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#295] [Notes]
# ScriptReference/Application-isEditor

https://docs.unity3d.com/ScriptReference/Application-isEditor.html

**Comment by joonjoonjoon** [comment]
If you are using the device similator package, `Application.isEditor` will return false when the simulator window is open or docked.  
Observed in Unity 2020.1.17f1 / Version 2.2.4-preview
